src/main/java/mate/academy/security/AuthenticationService.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.security;
+
+import mate.academy.exception.AuthenticationFailureException;
+import mate.academy.exception.RegistrationFailureException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationFailureException;
+
+    User register(String email, String password) throws RegistrationFailureException;
```

Comment:
unchecked exceptions shouldn't be thrown as here.
```suggestion
    User login(String email, String password);

    User register(String email, String password);
```